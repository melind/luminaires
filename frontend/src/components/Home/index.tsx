import React from 'react';
import userAPI from '../../services/userAPI';
import './index.css';

const Home = () => {
    
userAPI.home();

    return (
        <div className="home">
           zqeftesrgestrtresgrsebgtrs
           ergsetrgtrerhe
           egereg
        </div>
    )
}
export default Home;