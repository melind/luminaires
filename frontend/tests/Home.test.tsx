import React from 'react';
import renderer from 'react-test-renderer'
import { shallow, mount} from 'enzyme';
import Home from '../src/components/Home';

/* test de rendu*/
describe('Home snapshot', () => {
it('Home snapshot', () => {
  const tree = renderer
    .create(<Home />).toJSON();//use render() of component
  expect(tree).toMatchSnapshot();
});
/* test de fonctionnalitÃ©*/
 /* unit test*/

 it('home render one component', () => {
     const wrapper = shallow(<Home optInValue={true} authenticated={true}/>);
     expect(wrapper.length).toEqual(1);  // to have one Component
  }); 


});