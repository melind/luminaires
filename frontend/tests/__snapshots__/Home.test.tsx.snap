// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App should render correctly 1`] = `
<div
  className="home"
>
  zqeftesrgestrtresgrsebgtrs ergsetrgtrerhe egereg
</div>
`;

exports[`Home snapshot Home snapshot 1`] = `
<div
  className="home"
>
  zqeftesrgestrtresgrsebgtrs ergsetrgtrerhe egereg
</div>
`;
